content/concepts/consensus-network/amendments/known-amendments.md

```diff
@@ -1040,6 +1100,25 @@ Allows pre-authorization of accounting relationships (zero-balance trust lines)
 With this amendment enabled, a `TrustSet` transaction with [`tfSetfAuth` enabled](trustset.html#trustset-flags) can create a new [`RippleState` ledger object](ripplestate.html) even if it keeps all the other values of the `RippleState` node in their default state. The new `RippleState` node has the [`lsfLowAuth` or `lsfHighAuth` flag](ripplestate.html#ripplestate-flags) enabled, depending on whether the sender of the transaction is considered the low node or the high node. The sender of the transaction must have already enabled [`lsfRequireAuth`](accountroot.html#accountroot-flags) by sending an [AccountSet transaction](accountset.html) with the [`asfRequireAuth` flag enabled](accountset.html#accountset-flags).
 
 
+## XRPFees
+[XRPFees]: #xrpfees
+
+| Amendment    | XRPFees |
+|:-------------|:--------|
+| Amendment ID | 93E516234E35E08CA689FA33A6D38E103881F8DCB53023F728C307AA89D515A7 |
+| Status       | In Development |
+| Default Vote (Latest stable release) | No |
+| Pre-amendment functionality retired? | No |
+
+Simplifies transaction cost calculations to use XRP directly rather than calculating indirectly in "fee units" and translating the results to XRP. Updates all instances of "fee units" in the protocol and ledger data to be drops of XRP instead, including:
```

Comment:
I want @ximinez to look at this description.  The amendment was recently made backward compatible with the API, so there might not be any replaced fields, only additional fields.  I was not part of the code review, so I have no actual details.

Reply:
Asked him on Slack and he says, "Yeah, that looks right."

Reply:
With the fix, there are no RPC API changes. The ledger entry and transaction field changes described are correct.